src/index.html

```diff
@@ -53,8 +53,8 @@ <h2 id="sidebarName">Name Name</h2>
         <section tabindex="0" class="user-friends">
           <h2 id="sidebarName">FRIENDLY STEP CHALLENGE!</h2>
           <img src="./images/person walking on path.jpg" alt="A person's legs walking down a path" class="sidebar-header-challengeImage"></img>
-          <p tabindex="0">WINNER: <span id="bigWinner" tabindex:"0">PLACEHOLDER</span></p>
-            <ul tabindex="0" class="card-vertical-list" id="friendList" aria-label="Friend List">
+          <p id="bigWinner" tabindex="0"></p>
+            <ul tabindex="0" class="card-vertical-list" id="loserList" aria-label="Friend List">
```

Comment:
card-vertical-list is making all of these lists italicized. we'll have to decide if we want that in the final product